eras/alonzo/impl/src/Cardano/Ledger/Alonzo/TxBody.hs

```diff
@@ -514,8 +503,8 @@ instance
           (2, "fee")
         ]
 
-initial :: AlonzoTxBodyRaw era
-initial =
+basicAlonzoTxBodyRaw :: AlonzoTxBodyRaw era
+basicAlonzoTxBodyRaw =
```

Comment:
`emptyAlonzoTxBodyRaw`?  :bicyclist: :house:

Reply:
Sure, why not. Will change

Reply:
Done